OpenUpSA/wazimap-ng
# Feature/add url to metadata

## Description
Added a url field to the dataset metadata object

## Related Issue
N/A

## How to test it locally
Navigate to the dataset update page in admin. You will see a new url field there. 
Also, test the api with something like: http://localhost:8000/api/v1/profiles/8/geographies/KZN/?format=json, you should see a url entry in the api response

## Changelog

### Added
url field
appropriate migrations

### Updated
indicator data serializer

### Removed


## Checklist


### Pull Request


### Commits


### Code Quality


### Testing
